File: Lehrverwaltung/server/src/main/java/tie/lehrlinge/lehrverwaltung/server/model/dto/CreateUserDTO.java
package tie.lehrlinge.lehrverwaltung.server.model.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class CreateUserDTO {

  private String firstname;
  private String lastname;
  private String username;
  private String password;
  private String role;
}

File: Lehrverwaltung/client/src/app/core/entities/dtos/CreateUserDTO.ts
export class CreateUserDTO {
  firstname: string;
  lastname: string;
  username: string;
  password: string;
  role: string;

  constructor(firstname: string, lastname: string, username: string, password: string, role: string) {
    this.firstname = firstname;
    this.lastname = lastname;
    this.username = username;
    this.password = password;
    this.role = role;
  }
}
